upload/system/library/Packetery/Vendor/VendorRepository.php

```diff
@@ -6,6 +6,18 @@
 
 class VendorRepository {
 
+	const INTERNAL_VENDORS = [
```

Comment:
Chybí překlady.  To pole bych sestavil  takto 'id'=> 'zpoint', 'name'=> 'Výdejní místa',  countries=> ['cz', 'sk', 'ro', 'hu].  Bude to přehlednější a nemusíš několikrát opakovat názvy a idčka

Reply:
místo internal_vendors bych zvolil spíš název "packeta_vendors".